docs/UserGuide.md

```diff
@@ -33,95 +39,143 @@
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
+
+<br><img src="images/Help.jpeg" width="600"><br>
+
+#### Adding a task in CMDo: `add`
+Adds a task to CMDo <br>
+Format: `add <details> <day> <time> /<priority> -<tag>` 
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
- 
-> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
-> items with `...` after them can have multiple instances. Order of parameters are fixed. 
+> You can type anything in details. It must not end with `by`, `on`, `before` or `at`.
 > 
-> Persons can have any number of tags (including 0)
+> For time, typing `tml`, `tmr`, `tomorrow` will still be recognised as tomorrow.
```

Comment:
If your team is not looking at NLP you don't have to worry too much about this ~ however you can still provide the keywords specified above! it would be nice to have them!